specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,661 @@
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Terminate a Container Apps Job": {
+            "$ref": "./examples/ContainerAppsJob_Terminate.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Configuration": {
+      "description": "Non versioned Container Apps Job configuration properties that define the mutable settings of a Container apps job",
+      "type": "object",
+      "required": [
+        "activeDeadlineSeconds",
+        "crashLoopRetryLimit",
+        "triggerType"
+      ],
+      "properties": {
+        "secrets": {
+          "description": "Collection of secrets used by a Container app",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/Secret"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "activeDeadlineSeconds": {
+          "description": "Maximum duration of the job.",
+          "format": "int32",
+          "type": "integer"
+        },
+        "replicaRetryLimit": {
+          "format": "int32",
+          "description": "Maximum duration of the job.",
+          "type": "integer"
+        },
+        "manualTriggerConfig": {
+          "type": "object",
+          "required": [
+            "completions",
+            "parallelism"
+          ],
+          "properties": {
+            "completions": {
+              "description": "Number of pods that needs to be successfully\ncompleted for a job to be complete.",
+              "format": "int32",
+              "type": "integer"
+            },
+            "parallelism": {
+              "description": "Number of parallel jobs that can run at a given\ntime.",
+              "format": "int32",
+              "type": "integer"
+            }
+          }
+        },
+        "scheduleTriggerConfig": {
+          "required": [
+            "completions",
+            "cronExpression",
+            "parallelism"
+          ],
+          "type": "object",
+          "properties": {
+            "completions": {
+              "description": "Number of pods that needs to be successfully\ncompleted for a job to be complete.",
+              "format": "int32",
+              "type": "integer"
+            },
+            "cronExpression": {
+              "description": "Cron formatted repeating schedule (\"* * * * *\")\nfor cronjobs.",
+              "type": "string"
+            },
+            "parallelism": {
+              "description": "Number of parallel jobs that can run at a given\ntime.",
+              "format": "int32",
+              "type": "integer"
+            }
+          }
+        },
+        "registries": {
+          "description": "Collection of private container registry credentials for containers used by the Container app",
```

Comment:
"Collection of private container registry credentials for containers used by the Container app **job**"